Makefile

```diff
@@ -98,6 +98,10 @@ thrift: idl-submodule thrift-image
 	$(THRIFT) -o /data --gen go:$(THRIFT_GO_ARGS) --out /data/$(THRIFT_GEN_DIR) /data/idl/thrift/sampling.thrift
 	$(THRIFT) -o /data --gen go:$(THRIFT_GO_ARGS) --out /data/$(THRIFT_GEN_DIR) /data/idl/thrift/zipkincore.thrift
 	@echo Generate TChannel-Thrift bindings
+	# tchan doesn't support oneway, so to get collector signature, we need to modify the jaeger.thrift file
```

Comment:
I was just looking at the IDL. The oneway is defined on the Agent interface, but we already have `agent.thrift`. I think a cleaner change would be to add the jaeger-based emit method into the existing agent interface (it will need an import fix similar to L96), and the `jaeger.thrift` will work for tchannel's thrift-gen without changes